### ISSUE TITLE
Add image for PHP 8

### ISSUE THREAD

**Issue by wmortada** [michaelmcandrew/civicrm-buildkit-docker#71]
Now that CiviCRM [officially supports PHP 8](https://lab.civicrm.org/dev/core/-/issues/2201), I think we should add an image that runs PHP 8 to this repository.

**Comment by michaelmcandrew** [comment]
Yes - agreed. See https://lab.3sd.io/tools/civicrm-buildkit-docker/-/commit/d6686f5b5a09981ea6443bfe75c2cfbfd997ce8c. Just seeing what happens / if it will actually build...

**Comment by wmortada** [comment]
Just tested this locally and it works for me.

I'm getting a warning for the blank line in the Dockerfile though. This line should be deleted: [publish/templates/civicrm/Dockerfile.twig#L62](https://lab.3sd.io/tools/civicrm-buildkit-docker/-/blob/f9ca212d35a14feedabbb44c0dc72d2a8ba88eb3/publish/templates/civicrm/Dockerfile.twig#L62)

``` 
vim \
  zip \
# Delete me!
  && rm -r /var/lib/apt/lists/*
```
